### PR TITLE
Blog updates for release v1.3

### PR DESCRIPTION
**Pull request by PARitter**
Updates to the blog to delete the default page and install v1.3 release notes

### CODE REVIEW

File: src/pages/blog/Release-1.3.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+author: Phil Ritter
+date: 2020-10-15T09:00:00.000Z
+category:
+  - label: Release Notice
+    id: Release-A7fnZYrE1
+---
+# Magma v1.3 Release Notes
+
+
+This new major release for Magma ships with some great new features and enhanced support for the FWA use-case with federation. See therelease page (https://magma.github.io/magma/versions) and the https://fb.quip.com/UwaTAytJ49nQ for more information.
```
Comment (jbryce) [inline src/pages/blog/Release-1.3.md:13]: Typo: "therelease"

Comment (jbryce) [inline src/pages/blog/Release-1.3.md:13]: This fb.quip.com link seems to be some kind of FB internal link.

File: src/pages/blog/Release-1.3.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+
+
+## _New Features_
+
+### CDR Exporting
+
+This release add support for exporting Charging Data Records (CDR)  based on the IPFix (IP Flow Information Export) protocol. Magma provides the following additional fields: imsi, msisdn, apn_name, apn_mac, pdp_start_time, app_name.
```
Comment (jbryce) [inline src/pages/blog/Release-1.3.md:21]: Typo: "add" should be "adds"

File: src/pages/blog/Release-1.3.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+
+### SMS delivery service (SMSd)
+
+This new feature was added to support delivery of mobile-terminated SMS (i.e., receive-only SMS) to allow operators to deliver alerts to users. We have introduced a new mode of operation for the MME, SMS_ORC8R. This is essentially “SMS-in-MME” mode in the 3GPP specification, with delivery coordinated by the orc8r. SMS_ORC8R mostly re-uses infrastructure for the SGs interface used for CSFB support. Magma provides a REST API that can be utilized by partner serverices to send such messages to particular subscribers.
```
Comment (jbryce) [inline src/pages/blog/Release-1.3.md:26]: Typo: no comma after i.e.
Typo: "serverices" -> "services"

File: src/pages/blog/Release-1.3.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+## _Notable Fixes_
+
+### NMS
+
+* Fixed bugs around adding Federation Fixed wireless(feg_lte) networks.
+
+## _Known Issues_
+
+*
```
Comment (jbryce) [inline src/pages/blog/Release-1.3.md:73]: Appears to just have an empty bullet. Should we remove this section for now if there are no current Known Issues?

**Review by jbryce — CHANGES_REQUESTED** [review]
Great first post! I flagged a few typos and a couple of questions. I'm also going to make a comment in the conversation and flag Jimmy about a few set up questions.

**Comment by jbryce** [comment]
Do we need to update the nav to include the Blog in the nav @jimmytipit ? Do we want to do that in this PR or a follow-on PR.

Also, @PARitter I would suggest renaming the file to src/pages/blog/magma-release-1.3.md because I think it will help with search engines associating that title through the URL.

Finally, when you make these changes, you'll need to commit with the -S and if you force-push it will overwrite the commits and should clear up the DCO error: https://github.com/magma/community/blob/main/CONTRIBUTING.md#developers-certificate-of-origin

**Comment by PARitter** [comment]
Thanks Jonathan. Appreciate the review.

I think the PR to unmask the blog should be separate.

Phol
________________________________

**Comment by jimmytipit** [comment]
The nav can be modified after the blog post goes through. We've hidden it thus far since there are no posts. I'd advise if we could get a couple of posts up, it would be better. But can unveil at any time.

Cheers,
Jimmy

**Comment by PARitter** [comment]
@jbryce - I corrected the typo's and other changes as requested.  I did signoff the commits (--signoff) but the DCO check still fails.  Not sure what is happening there.  Any suggestions appreciated.

**Comment by jimmytipit** [comment]
Hi Phil - It appears that not all of the commits were signed off.  I know it's kind of a pain b/c I've been through it ;)  But can I suggest squashing your commits?   Then you can sign off and remove the commits you don't need.

https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec
https://www.internalpointers.com/post/squash-commits-into-one-git

image

**Comment by jimmytipit** [comment]
BTW - an alternate if you dont' want to deal with that is to close the PR, copy the work you've done, then checkout a fresh repo and do it all in one commit.  Sometimes that's easier than trying to wrestle with git. Personally I hate the DCO ;) But such is life.

**Comment by PARitter** [comment]
Closing this pull request and starting over to get DCO signature correct.

**Pull request closed without merging**